packages/lib-user/src/components/shared/Select/theme.js

```diff
@@ -34,10 +35,19 @@ const selectTheme = {
     icons: {
       color: {
         dark: 'white',
-        light: 'dark-5',
+        light: 'dark-5'
       },
       margin: { left: '0', right: 'xsmall' }
     },
+    listbox: {
+      extend: `
+        & > button {
+          &[aria-selected='true'] {
+            text-decoration: underline;
```

Comment:
Sean's request to have currently selected options more visually obvious in all shared Select components.